manifests/b/BethesdaSoftworks/Bethesda/1.75.0.0/BethesdaSoftworks.Bethesda.locale.en-US.yaml

```diff
@@ -5,6 +5,7 @@ PackageVersion: 1.75.0.0
 PackageLocale: en-US
 Publisher: Bethesda Softworks
 PackageName: Bethesda.net Launcher
+Url: https://bethesda.net/en/game/bethesda-launcher

```

Comment:
```suggestion
PackageUrl: https://bethesda.net/en/game/bethesda-launcher
```

Reply:
You also need to change `PackageVersion: 1.75.0.0` to `PackageVersion: 1.75.0` (and update the version number folder) to match the version number it shows in Control Panel.
![image](https://user-images.githubusercontent.com/11600822/120536846-7baba500-c3dc-11eb-8c19-d3a84faf5c73.png)